charts/rancher-monitoring/rancher-monitoring-crd/100.0.0+up16.6.0/values.yaml

```diff
@@ -7,5 +7,6 @@ global:
     systemDefaultRegistry: ""
 
 image:
+  # keep this inline with whatever Rancher's chart uses to avoid missing tag in rancher-images.txt
```

Comment:
```suggestion
  # keep this inline with both the primary monitoring chart and the rancher shell setting to avoid missing tag in rancher-images.txt
```